app/views/mentors/applications/show.slim

```diff
@@ -8,14 +8,22 @@ h2
   small For #{link_to @application.project_name, project_path(@application.project_id)} #{icon('star') if @application.first_choice?}
 
 div.panel.panel-default
-  div.panel-heading #{ApplicationDraft.human_attribute_name :why_selected_project1}
+  div.panel-heading
+    - if @application.first_choice?
```

Comment:
Can we make this a helper method?

Reply:
Hm, I think I don't really get what you have in mind 🙈 

I'll just interpolate the key - that also removes the conditional and drys up the view. Fine?

Reply:
tbh I don't know what I meant with this, either :see_no_evil:   Ignore it :)